examples/cluster/plot_digits_linkage.py

```diff
@@ -35,15 +35,16 @@
 
 np.random.seed(0)
 
+
 def nudge_images(X, y):
     # Having a larger dataset shows more clearly the behavior of the
     # methods, but we multiply the size of the dataset only by 2, as the
     # cost of the hierarchical clustering methods are strongly
     # super-linear in n_samples
-    shift = lambda x: ndimage.shift(x.reshape((8, 8)),
-                                  .3 * np.random.normal(size=2),
-                                  mode='constant',
-                                  ).ravel()
+    def shift(x): return ndimage.shift(x.reshape((8, 8)),
```

Comment:
Maybe add a line break

Reply:
I thought we told flake8 to ignore that kind of thing?

in setup.cfg we ignore:
```
    E731,  # do not assign a lambda expression, use a def
```

Reply:
Autopep8 did its magic here. I reverted the change and just fixed this part manually